utils/build/docker/nodejs/nextjs/src/app/flush/route.js

```diff
@@ -0,0 +1,39 @@
+import { promisify } from 'util'
+import { NextResponse } from 'next/server'
+
+export const dynamic = 'force-dynamic'
+
+export async function GET (request) {
+  await flush()
```

Comment:
the nextjs variant is failing because the request to this endpoint is failing. Idk why, it looks like it's throwing an empty error ?

Reply:
profiling doesn't work with nexjts apparently